jobs.md

```diff
@@ -0,0 +1,32 @@
+# Jobs
+
+Come and work with us!
+
+## Upstream team
+
+Interested in working on open source and containers at a leading hyper-scale cloud? You can help build the future of Kubernetes at Microsoft!  
+
+### Engineering
+#### Secure supply chain
+- [Software Engineer II, Atlanta, remote](https://careers.microsoft.com/us/en/job/1210002/Software-Engineer-II)
+- [Senior Software Engineer, Atlanta, remote](https://careers.microsoft.com/us/en/job/1309571/Senior-Software-Engineer)
```

Comment:
`Atlanta or US remote`?

Reply:
This means it's opened for Atlanta but can be US remote. Does that help?